config/trees.yaml

```diff
@@ -351,232 +407,263 @@ build_configs:
-    tree: collabora-chromeos-kernel
+  collabora-chromeos-kernel_for-kernelci: &collabora
+    <<: *base-standard
     branch: 'for-kernelci'
+    tree: collabora-chromeos-kernel
 
   collabora-next_for-kernelci:
+    <<: *collabora
     tree: collabora-next
-    branch: 'for-kernelci'
 
   efi:
+    <<: *base-standard
     tree: efi
     branch: 'next'
 
-  kernelci_staging-mainline:
+  hyperv-next: &hyperv
+    <<: *base-standard
+    tree: hyperv
+    branch: 'hyperv-next'
+    architectures:
+      - x86_64
+
+  hyperv-fixes:
+    <<: *hyperv
+    branch: 'hyperv-fixes'
+
+  kernelci: &kernelci
+    <<: *base-standard
     tree: kernelci
+
+  kernelci_staging-mainline:
+    <<: *kernelci
     branch: 'staging-mainline'
 
   kernelci_staging-next:
-    tree: kernelci
+    <<: *kernelci
     branch: 'staging-next'
 
   kernelci_staging-stable:
-    tree: kernelci
+    <<: *kernelci
     branch: 'staging-stable'
 
   khilman:
+    <<: *base-arm
     tree: khilman
     branch: 'to-build'
 
   krzysztof:
+    <<: *base-arm
     tree: krzysztof
     branch: 'for-next'
 
-  kselftest_fixes:
+  kselftest_fixes: &kselftest
+    <<: *base
     tree: kselftest
     branch: 'fixes'
 
   kselftest_next:
-    tree: kselftest
+    <<: *kselftest
     branch: 'next'
 
-  lee_backlight:
+  lee_backlight: &lee
+    <<: *base-standard
     tree: lee-backlight
     branch: 'for-backlight-next'
 
   lee_mfd:
+    <<: *lee
     tree: lee-mfd
     branch: 'for-mfd-next'
 
-  linusw_devel:
+  linusw: &linusw
+    <<: *base-arm
     tree: linusw
+
+  linusw_devel:
+    <<: *linusw
     branch: 'devel'
 
   linusw_fixes:
-    tree: linusw
+    <<: *linusw
     branch: 'fixes'
 
   linusw_for-next:
-    tree: linusw
+    <<: *linusw
     branch: 'for-next'
 
+  linux-pci:
+    tree: linux-pci
+    branch: 'next'
+    architectures:
+      - x86_64
+      - arm64
+      - arm
+
   mainline:
+    <<: *base
     tree: mainline
     branch: 'master'
 
-  media_fixes:
-    tree: media
+  media-committers_fixes: &media-committers
+    <<: *base-standard
+    tree: media-committers
     branch: 'fixes'
 
-  media_next:
-    tree: media
+  media-committers_next:
+    <<: *media-committers
     branch: 'next'
 
-  media-committers_fixes:
-    tree: media-committers
+  media_fixes: &media
+    <<: *base-standard
+    tree: media
     branch: 'fixes'
 
-  media-committers_next:
-    tree: media-committers
+  media_next:
+    <<: *media
     branch: 'next'
 
   mediatek_for_next:
+    <<: *base-arm
     tree: mediatek
     branch: 'for-next'
 
   net-next:
+    <<: *base
     tree: net-next
     branch: 'main'
 
   netdev-testing:
+    <<: *base
     tree: netdev-testing
     branch: 'https://netdev.bots.linux.dev/static/nipa/branches-hw.json'
 
-  next_master:
+  next_master: &next
+    <<: *base
     tree: next
-    branch: 'master'
 
   next_pending-fixes:
-    tree: next
+    <<: *next
     branch: 'pending-fixes'
 
   omap:
+    <<: *base-standard
```

Comment:
Please correct me if I'm wrong - does x86_64 build make sense for this tree? Shouldn't it be `base-arm`?

Reply:
Good catch. Makes sense.

Reply:
Done.